raiden/raiden_service.py

```diff
@@ -615,28 +596,37 @@ def _initialize_transactions_queues(self, chain_state: ChainState):
                     else:
                         raise
 
+    def _initialize_payment_statuses(self, chain_state: ChainState):
+        """ Re-initialize targets_to_identifiers_to_statuses. """
+
+        with self.payment_identifier_lock:
+            for task in chain_state.payment_mapping.secrethashes_to_task.values():
+                if not isinstance(task, InitiatorTask):
+                    continue
+
+                # Every transfer in the transfers_list must have the same target
+                # and payment_identifier, so using the first transfer is
+                # sufficient.
+                transfer = next(task.manager_state.initiator_transfers.values())
```

Comment:
@hackaugusto You'd have to `iter(values())`.

Reply:
gaaahhhh, I thought `.values()` was an iterator already